src/server/routes/rest.js

```diff
@@ -155,7 +155,11 @@ router.post('/enrichment', (req, res) => {
 // Expose a rest endpoint for emap
 router.get('/emap', (req, res) => {
   const pathwayIdList = req.query.pathwayIdList.split(/\s+/);
-  res.json(generateGraphInfo(pathwayIdList));
+  try {
+    res.json(generateGraphInfo(pathwayIdList));
+  } catch (err) {
+    res.json(err.message);
```

Comment:
What is the http status when you return an error message?

Reply:
`info:    GET /api/emap/?pathwayIdList=GO:0043525%20GO:0043523 304 1.533 ms - -`

Reply:
you should probably send a different status.
https://expressjs.com/en/api.html#res.sendStatus

you should probably send a 4XX error, possibly 400, or maybe 500
http://www.restapitutorial.com/httpstatuscodes.html#